src/views/product-tutorials-view/helpers/__tests__/detect-and-reformat-learn-url.test.ts

```diff
@@ -166,6 +168,12 @@ describe('detectAndReformatLearnUrl', () => {
 				input: '/tutorials/waypoint/aws-ecs',
 				expected: '/waypoint/tutorials/deploy-aws/aws-ecs',
 			},
+			{
+				input:
+					'/tutorials/consul/gossip-encryption-secure?utm_source=consul.io&utm_medium=docs',
```

Comment:
💭 I wonder if they should remove these query params from the docs source anyways since they aren't necessarily relevant to devdot anymore?

Reply:
Not saying we shouldn't support this case, just thinking outloud for content